src/open_inwoner/accounts/models.py

```diff
@@ -354,6 +354,10 @@ def get_full_name(self):
     def get_short_name(self):
         return self.first_name
 
+    @property
```

Comment:
What do you find confusing about `call_name`? It doesn't replicate `display_name` because it returns either the `display_name` (if it's set) or the `first_name`. Am I missing something?

Reply:
Now I get it. I stumbled over this myself and couldn't come up with a great name, since "display_name" would already be appropriate.